include/dali/core/gpu_utils.h

```diff
@@ -16,17 +16,27 @@
 #define DALI_CORE_GPU_UTILS_H_
 
 #include <cuda_runtime.h>
+#include "dali/core/cuda_utils.h"
 
 namespace dali {
 
 class DeviceGuard {
  public:
+  DeviceGuard() {
```

Comment:
what's the point of this?

Reply:
Now? Hmm... i don't think there's much reason to keep it; we can use the other one.